pkg/apierrors/apierrors.go

```diff
@@ -205,6 +206,13 @@ func NewGithubRepositoryNotFoundError() *APIError {
 	}
 }
 
+func NewGithubReleaseCreationError() *APIError {
```

Comment:
What category is it? )

Reply:
Good point. I have realized I don't yet know if the action will be executed asynchronously. So, for now, I will just return the error. Once I know how the function is used in the service, I will handle the service error accordingly.